build-tests/api-extractor-scenarios/etc/test-outputs/functionOverload/public-rollup.d.ts

```diff
@@ -0,0 +1,16 @@
+
+/* Excluded from this release type: combine */
+
+/**
+ * @public
+ */
+export declare class Combiner {
+    /* Excluded from this release type: combine */
```

Comment:
This may be out of scope for this change, but it'd be good to include which overload was excluded, or indicate why it was excluded.

The output here is a little confusing because `combine` is clearly there.

Reply:
Agreed, but that information isn't easily available in this context.  We'd need something like [ApiModelGenerator._getOverloadIndex()](https://github.com/microsoft/rushstack/blob/e792300b15cfa073077ed47bfc09798ebef4174a/apps/api-extractor/src/generators/ApiModelGenerator.ts#L798), or ideally a way to look up the corresponding `ApiItem`, but both of these require maintaining a cached map.

So I'd say it's out of scope for this PR.